packages/ember-views/lib/views/view.js

```diff
@@ -763,14 +763,20 @@ var View = CoreView.extend(
 
     @property layout
     @type Function
-  */
-  layout: computed('layoutName', function(key) {
-    var layoutName = get(this, 'layoutName');
-    var layout = this.templateForName(layoutName, 'layout');
+    */
+  layout: computed('layoutName', {
```

Comment:
i don't believe so. I am not sure why `defaultLayout` still exists, is there some piece of functionality I am missing?